Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.yml

```diff
@@ -0,0 +1,407 @@
+category: Vulnerability Management
+commonfields:
+  id: AzureRiskyUsers
+  version: -1
+configuration:
+- display: Client ID
+  name: client_id
+  required: true
+  type: 4
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+description: Azure Risky Users provides access to all at-risk users and risk detections
+  in Azure AD environment.
+display: Azure Risky Users
+name: AzureRiskyUsers
+script:
+  commands:
+  - deprecated: false
+    description: Tests the connectivity to Azure.
+    execution: false
+    name: azure-risky-users-auth-test
+  - deprecated: false
+    description: Run this command to start the authorization process and follow the
+      instructions in the command results.
+    execution: false
+    name: azure-risky-users-auth-start
+  - deprecated: false
+    description: Run this command to complete the authorization process. Should be
+      used after running the azure-risky-users-auth-start command.
+    execution: false
+    name: azure-risky-users-auth-complete
+  - deprecated: false
+    description: Run this command if for some reason you need to rerun the authentication
+      process.
+    execution: false
+    name: azure-risky-users-auth-reset
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: Risk State to retrieve. If not specified, all states will be retrieved.
+      isArray: false
+      name: risk_state
+      predefined:
+      - atRisk
+      - confirmedCompromised
+      - remediated
+      - dismissed
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: Limit of results to retrieve.
```

Comment:
Specify the default in the description, here and in other descriptions (for example, page number).